### PR TITLE
Prettify sequential test output

### PR DESCRIPTION
**Pull request by yenWu**
I wonder to see test units in good order ; )

### CODE REVIEW

File: tests/runner.py [hunk 1]
```diff
@@ -24,7 +24,9 @@
 
 
 def find_all_tests(excludes=[]):
-    tests = list(map(lambda p: p.strip('/').split('/')[-1], glob.glob('tests/*_[0-9]*')))
```
Comment (mlouielu) [inline tests/runner.py]: You can do like this:

`tests = list(map(lambda p: p.strip('/').split('/')[-1], sorted(glob.glob('tests/*_[0-9]*'))))`

Comment (mlouielu) [reply tests/runner.py]: Adding `sorted` to make it sort and one-line :)

Comment (yenWu) [reply tests/runner.py]: Brilliant! That's what truly prettification. I've done that.

**Comment by mlouielu** [comment]
hmmmmm, CI was running Python2, not 3, it should be fixed.

**Comment by yenWu** [comment]
That's right. I'm getting familiar with travis-CI and trying to fix this issue.

Moreover, I notice that the version of python3(ver-3.4) may be too low to use `subprocess.run()`. The version has to be above 3.5.

**Comment by yenWu** [comment]
@mlouielu , I already fix travis-ci in a coarse way. And I'm ready to merge this branch.

**Pull request merged**